### PR TITLE
Feature/11 interests

### PR DESCRIPTION
**Pull request by eddiejaoude**

### CODE REVIEW

File: src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java [hunk 1]
```diff
@@ -12,5 +14,14 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
+    @NotNull
+    @Size(min=2, max=32)
     private String name;
+
+    Skill() {
```
Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java:21]: Does this need to be declared?

Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java:21]: Nevermind mobile view

Comment (eddiejaoude) [inline src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java:21]: Yeah it does, its needed for Reflection for JPA - I found it odd too

Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java:21]: Surely not having visibility on the constructor would make it private though..   Weird

File: src/main/java/com/empowerhack/family/memberservice/FamilyMemberServiceApplication.java [hunk 2]
```diff
@@ -9,4 +18,61 @@
     public static void main(String[] args) {
         SpringApplication.run(FamilyMemberServiceApplication.class, args);
     }
+
+    @Bean
```
Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/FamilyMemberServiceApplication.java:22]: Seeds in the application?

File: src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java (hunk 1 above)
Comment (eddiejaoude) [inline src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java:21]: Yeah it is private with no visibility, through reflection it must have access. Java magic :p

File: src/main/java/com/empowerhack/family/memberservice/FamilyMemberServiceApplication.java (hunk 2 above)
Comment (eddiejaoude) [inline src/main/java/com/empowerhack/family/memberservice/FamilyMemberServiceApplication.java:22]: Yeah, temporary solution just for playing with while using **In-memory** Database

File: src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java (hunk 1 above)
Comment (brettminnie) [inline src/main/java/com/empowerhack/family/memberservice/Entity/Skill.java:21]: What ever happened to controlling code.

Damn kids, get off my lawn!!!

**Pull request merged**